.changeset/clever-badgers-begin.md

```diff
@@ -0,0 +1,5 @@
+---
+"@effection/node": major
+---
+
+Let spawn shell be set, but default to process.env.shell which is a good default for Windows along with windowsHide.
```

Comment:
@jbolda While major is the right change, I think the changelog message is no longer needed.

Reply:
🤦‍♂️